sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/HttpTransportClient.java

```diff
@@ -251,6 +248,16 @@ protected GlobalEndpointManager getGlobalEndpointManager() {
         return this.globalEndpointManager;
     }
 
+    @Override
+    public IOpenConnectionsHandler getOpenConnectionsHandler() {
```

Comment:
Returning null here to not affect CI pipelines.

Reply:
what is the issue in the CI pipeline?